src/python/turicreate/test/test_image_classifier.py

```diff
@@ -19,12 +19,14 @@
     _raise_error_if_not_sframe,
     _raise_error_if_not_sarray,
 )
+from turicreate.toolkits.image_analysis.image_analysis import MODEL_TO_FEATURE_SIZE_MAPPING, get_deep_features
 
 from . import util as test_util
 
 import coremltools
 import numpy as np
 import turicreate as tc
+from array import array
```

Comment:
It doesn't look like you're using this import in this file.

Reply:
You are right, this will be removed